tests/deferred_yield_higher/Makefile

```diff
@@ -1,3 +1,5 @@
 include ../Makefile.tests_common
 
+BOARD_WHITELIST := native
```

Comment:
Why not

```suggestion
FEATURES_BLACKLIST += arch_avr8 arch_msp430
```

Reply:
hm, I'm not sure about the others. but yes, that's nicer

Reply:
done, I've also added esp, mips and riscv, until they're confirmed to work.

Reply:
don't we have esp on CI?
I have esp on my desk, I can do the test just now.

Reply:
#### esp8266

```
/home/benpicco/dev/RIOT/dist/tools/pyterm/pyterm -p "/dev/ttyUSB3" -b "115200" --no-reconnect --noprefix --no-repeat-command-on-empty-line 
Twisted not available, please install it if you want to use pyterm's JSON capabilities
Connect to serial port /dev/ttyUSB3
Welcome to pyterm!
Type '/exit' to exit.
READY
s
START
main(): This is RIOT! (Version: 2021.10-devel-380-ge4b3a)
1. post yield
2. second thread scheduled
3. post irq enable
```

works

#### esp32

```
/home/benpicco/dev/RIOT/dist/tools/pyterm/pyterm -p "/dev/ttyUSB3" -b "115200" --no-reconnect --noprefix --no-repeat-command-on-empty-line 
Twisted not available, please install it if you want to use pyterm's JSON capabilities
Connect to serial port /dev/ttyUSB3
Welcome to pyterm!
Type '/exit' to exit.
READY
s
START
main(): This is RIOT! (Version: 2021.10-devel-380-ge4b3a)
1. post yield
2. second thread scheduled
3. post irq enable
```

works

Reply:
Doesn't esp need to be whiteliseted now @kaspar030 ?

Reply:
thanks @benpicco! I've removed the esp blacklist.